metricflow-semantics/metricflow_semantics/mf_logging/pretty_print.py

```diff
@@ -55,6 +57,10 @@ def _handle_sequence_obj(self, list_like_obj: Union[list, tuple], remaining_line
         elif isinstance(list_like_obj, tuple):
             left_enclose_str = "("
             right_enclose_str = ")"
+        elif isinstance(list_like_obj, set) or isinstance(list_like_obj, frozenset):
+            left_enclose_str = f"{type(list_like_obj).__name__}("
```

Comment:
Note, `{}` is more common for sets but 1. they look ugly and 2. `{` indicates a format substring so meh.